### DISCUSSION TITLE
CAD

### DISCUSSION THREAD

**Discussion by blobfish** [vsg-dev/VulkanSceneGraph#994] [Show and tell]
My first VSG scene from an OpenCasCade model.
![firstVsgOcctModel](https://github.com/vsg-dev/VulkanSceneGraph/assets/1299651/1c6a9643-e288-4140-8f90-64f06b8ec94f)

**Comment by timoore** [comment]
That's very cool! I've been interested in OpenCascade for a while, but it seems quite opaque. Is your VSG code online?

**Comment by blobfish** [comment]
Opaque is a nice way to put it. I am just getting my feet wet with Vulkan/VSG so I hadn't pushed my repo anywhere. Because you asked, I pushed it to [gitlab](https://gitlab.com/blobfish/occtvsg) FYI: Once I know more about what I am doing, I will probably abandon this project and start something fresh. Master is currently broken as I was trying to figure out how to generate edges and I decided to give up on vsg::builder and create something from scratch.
